.env.example

```diff
@@ -1,2 +1,2 @@
-BASE_URL=http://localhost:3000
-BACKEND_URL=https://ah-jawans-backend.herokuapp.com
+REACT_APP_BASE_URL=https://ah-jawans-backend.herokuapp.com
```

Comment:
@shaluchandwani The values of the env variables must not be exact, the `.env.example` should just tell what values are expected. This is a security vulnerability on the project. an example would be `BACKEND_URL=project_backend_url` and in the readme one should told to get the variables from the dev team.